proto/p4/config/p4info.proto

```diff
@@ -21,13 +21,11 @@ import "google/protobuf/any.proto";
 package p4.config;
 
 message P4Info {
-  repeated HeaderField header_fields = 1;
```

Comment:
Yes, Antonin is correct - we should be able to do this in the p4c backend for the switch.  As we discussed on Friday, the p4c IR still contains all the header field data, it just never gets into P4Info.